x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/data_streams/resolution_table_cell.tsx

```diff
@@ -66,23 +73,61 @@ const getI18nTexts = (resolutionType?: DataStreamResolutionType) => {
         values: { resolutionType },
       }
     ),
+    recommendedActionTexts: {
+      readonly: {
+        text: i18n.translate(
+          'xpack.upgradeAssistant.esDeprecations.dataStream.recommendedActionReadonlyText',
+          {
+            defaultMessage: 'Recommended to set to read-only',
```

Comment:
```suggestion
            defaultMessage: 'Recommended: set to read-only',
```
For any recommended actions, adding Recommended: as a prefix followed by an action makes it clearer that this is a suggestion + action combination.

Reply:
I'm not sure if this also needs to be updated in other places in the code or only here in the default messages of `recommendedActionsTexts`.